helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -141,9 +147,10 @@ public int compare(TaskStatusEvent arg0, TaskStatusEvent arg1) {
       STRING_LIST_TYPE =
       new TypeReference<List<String>>() {};
 
-  private static final String DEPLOYMENT_GROUP_EVENTS_KAFKA_TOPIC = "HeliosDeploymentGroupEvents";
-  private static final DeploymentGroupEventFactory DEPLOYMENT_GROUP_EVENT_FACTORY =
-      new DeploymentGroupEventFactory();
+  public static final int MAX_ROLLING_OPERATION_HISTORY = 10;
```

Comment:
Can be private. While you're at it, a lot of the attrs above can also be private.